packages/lib-classifier/src/store/Classification.js

```diff
@@ -36,15 +37,12 @@ const ClassificationMetadata = types.model('ClassificationMetadata', {
 
 const Classification = types
   .model('Classification', {
-    annotations: types.map(types.union({
-      dispatcher: (snapshot) => {
-        const snapshotType = getType(snapshot)
```

Comment:
`getType` can't be run on a snapshot, only a tree node. This dispatcher works because we pass in a tree node as the snapshot. `getType` is the default dispatcher, so I've removed this function because it doesn't really do anything.